pythonfile/manifest_validator.py

```diff
@@ -0,0 +1,28 @@
+import json5
+
+import jsonschema
+from jsonschema import Draft4Validator
+from jsonschema import FormatChecker
+from jsonschema import validate
+
+
+class ManifestValidator:
+    def __init__(self, json_schema):
+        self.schema = json5.load(json_schema)
+        self.validator = Draft4Validator(self.schema, format_checker=FormatChecker())
+
+    def validate_manifest(self, json_manifest):
+        try:
+            manifest = json5.load(json_manifest)
+            result = self.validator.validate(manifest)
+            return result
+        except FileNotFoundError:
```

Comment:
How is this error possible, if `json_manifest` is a string? Can `json5.load()` also accept filenames?
Or is this a left-over?